client/src/components/DownloadOption.js

```diff
@@ -62,7 +63,7 @@ export const DownloadOption = ({ computedFile, handleSelectFile }) => {
         </Box>
       </Box>
       <Box gridArea="footer" margin={{ top: 'medium' }}>
-        <Box>
+        <Grid columns={['auto', 'auto']} gap="large">
```

Comment:
Add responsiveness to the UI to support mobile view here - please refer to the screenshots!

```suggestion
// import the useResponsive hook in this file
import { useResponsive } from 'hooks/useResponsive' 

export const DownloadOption = ({ computedFile, handleSelectFile }) => {
  const { responsive } = useResponsive() // destructure the 'responsive' method here
  
// use this in the columns prop and pass 'small' and 'big' values 
  <Grid columns={responsive('1', '1/2')} gap="large">
  ```
  
For large devices:

![Screenshot 2024-10-01 at 2 15 28 PM](https://github.com/user-attachments/assets/99205959-97bb-4b02-a03f-9c06610faf20)

On mobile, these buttons are stacked:

![Screenshot 2024-10-01 at 2 15 20 PM](https://github.com/user-attachments/assets/5acb8ded-1da2-4985-9e58-4c4a319ec77f)